geometry_msgs/Point[] Position_XYZ
uint16 center_pixel_x
uint16 center_pixel_y
uint8 counter
uint8 color_type
float32 angle
